CMakeLists.txt

```diff
@@ -53,6 +53,9 @@ if (IS_DIRECTORY "${PROJECT_SOURCE_DIR}/.git")
     file(
       COPY scripts/pre-commit
       DESTINATION "${PROJECT_SOURCE_DIR}/.git/hooks")
+    file(
+      COPY scripts/commit-msg
```

Comment:
`COPY` takes multiple files so this can just be `COPY scripts/pre-commit scripts/commit-msg`

Reply:
Simplified this